src/components/ratings_reviews/Reviews.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import ReviewListEntry from './ReviewListEntry';
+
+function Reviews() {
+  return (
+    <div
+      style={{
+        display: 'flex',
+        flexDirection: 'column',
+        marginLeft: '0.2em',
+        border: 'solid 0px white',
+        padding: '0.2em 0em 0em 0em',
+        'border-radius': '5px',
+        justifyContent: 'center',
+        alignItems: 'center',
+      }}
+    >
+      <em>Reviews</em>
+      <select
+        style={{
+          display: 'flex',
+          flexDirection: 'row',
+          float: 'right',
+        }}
+      >
+        <option>Sort by...</option>
+        <option>Helpfulness</option>
+        <option>Newest</option>
+        <option>Relevance</option>
+      </select>
+      <br />
+      <ReviewListEntry />
+      <ReviewListEntry />
+      <button
+        type="submit"
```

Comment:
Same as my other button comment above -- would recommend switching to `type="button"` since the parent isn't a `<form>`